packages/backend/src/app/helper/engine-helper.ts

```diff
@@ -65,30 +65,29 @@ export const engineHelper = {
         }) as ExecutionOutput
     },
     async executeParseEvent(operation: ExecuteEventParserOperation): Promise<ParseEventResponse> {
-        const sandbox = sandboxManager.obtainSandbox()
+        const sandbox = await sandboxManager.obtainSandbox(apId())
```

Comment:
Any reason not to use `${pieceName}:${pieceVersion}` for `resourceId` in `executeParseEvent` as well?